token-locker/rust-toolchain

```diff
@@ -0,0 +1 @@
+stable-2021-12-02
```

Comment:
Let's use the latest Rust version

```suggestion
stable-2022-02-24
```

Reply:
https://github.com/aurora-is-near/rainbow-token-connector/commit/1f37b47966119d55597f6c40a90332ef66b130c1